Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/speedInput/index.tsx

```diff
@@ -1,14 +1,29 @@
-const SpeedInput = () => {
+interface Props {
```

Comment:
You define the Props interface here, but don't use it in line 6 as the type of your props